src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -1,11 +1,11 @@
 import { LightningElement, api, track, wire } from 'lwc';
 import { fireEvent } from 'c/pubsubNoPageRef';
-import { ShowToastEvent } from 'lightning/platformShowToastEvent';
-
-import { getObjectInfo, getPicklistValues } from 'lightning/uiObjectInfoApi';
-import { getRecord} from 'lightning/uiRecordApi';
+import { showToast } from 'c/utilTemplateBuilder';
```

Comment:
Using `c/utilTemplateBuilder` instead of creating the same `showToast` method in the RD2 Entry Form.